hw/formal/fpv_all

```diff
@@ -7,24 +7,26 @@
 # list all blocks
 #-------------------------------------------------------------------------
 declare -a blocks=(
+  "aes"
+  "alert_handler"
   "gpio"
   "rv_core_ibex"
   "rv_dm"
   "spi_device"
   "rv_timer"
   "uart"
   "hmac"
+  "i2c"
   "flash_ctrl"
   "usbuart"
   "usbdev"
-  "usb_fs_nb_pe"
   # TODO: bind files are deleted
   # "tlul_adapter_sram"
   # "tlul_socket_1n"
   # "tlul_socket_m1"
   # "sram2tlul"
   # "xbar_main"
-  #"top_earlgrey"
+  "top_earlgrey"
```

Comment:
The changes in this file should have been a separate commit (within this PR), since they are not really related to the other fusesoc-related changes.

Reply:
That is a good reminder, thanks Philipp for the detailed review. I will take it away.